lib/src/config/introduction.dart

```diff
@@ -35,13 +34,12 @@ class IntroductionPage {
   /// The values for [title] and [text] are optional in this, but will use a
   /// default translation key when built.
   ///
-  /// The [backgroundImage] is fully optional and if not provided will show the
-  /// [ThemeData.backgroundColor] as default.
+  /// The [decoration] is fully optional and if not provided will show the
+  /// [ThemeData.colorScheme.background] as default.
   IntroductionPage({
     this.title,
     this.text,
     this.graphic,
-    this.backgroundImage,
```

Comment:
If you are updating API with a breaking change, then the version should go up by 1 major version